deploy.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# -------- Build Settings -------- #
+# directory to deploy
+BUILD_DIR="./dist"
+# directory on target server
+DEPLOY_DIR="./latest"
+# use to install dependencies without disruption
+TEMP_DIR="./staged"
+# server to connect to
+TARGET="hanbot@hanbot.elviswolcott.com"
+# file to run
+ENTRY="index.js"
```

Comment:
You shouldn't specify this here. Just tell `node` (or `pm2`) to run the directory (ex. `node ~/hanbot`), and `package.json` will tell it what the entry point is.